moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -366,8 +367,18 @@ ompl_interface::ModelBasedPlanningContextPtr ompl_interface::PlanningContextMana
     }
   }
 
-  ModelBasedPlanningContextPtr context =
-      getPlanningContext(pc->second, boost::bind(&PlanningContextManager::getStateSpaceFactory2, this, _1, req), req);
+  // Check for forced use of JointModelStateSpace
```

Comment:
please expand this comment (just copy some of the PR description) because this is somewhat of a hack